enhancements/api-review/add-ClusterImagePolicy-and-ImagePolicy-for-Signature-Verification.md

```diff
@@ -145,6 +146,12 @@ type PolicyRootOfTrust struct {
 	PolicyType PolicyType `json:"policyType"`
 	// publicKey defines the root of trust based on a sigstore public key.
 	// +optional
+
+    // ImagePolicySecretRef references a Kubernetes Secret containing PolicyType data.
+    // The Secret should be of type 'Opaque' and include the necessary PolicyType information.
+    // +optional
+    ImagePolicySecretRef *SecretReference `json:"imagePolicySecretRef,omitempty"`
```

Comment:
I reviewed this PR with Harshal. There is a v1.Secret object, and this is a reference. I vote on keeping `ImagePolicySecretRef` so that it is clear that it just references a secret.

Reply:
> I reviewed this PR with Harshal. There is a v1.Secret object, and this is a reference. I vote on keeping ImagePolicySecretRef so that it is clear that it just references a secret.

/cc @saschagrunert are you okay with going back to `imagePolicySecretRef`?